common/src/main/java/org/apache/uniffle/common/config/RssBaseConf.java

```diff
@@ -129,6 +129,12 @@ public class RssBaseConf extends RssConf {
           .defaultValue(ClientType.GRPC)
           .withDescription("client type for rss");
 
+  public static final ConfigOption<Long> RSS_CLIENT_TYPE_GRPC_TIMEOUT =
+      ConfigOptions.key("rss.rpc.client.type.grpc.timeout")
+          .longType()
```

Comment:
Could we use the method checkValue to validate the config option?

Reply:
+1

Reply:
Hi @jerqi  @zuston 

Thanks for the prompt reply and I updated the part you mentioned.